components/HomePage.js

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect } from 'react'
+import styled from 'styled-components/native'
+import headerImage from '../assets/mountain.jpg';
+import { Button } from 'react-native'
+
+const HomePage = ({ navigation, route }) => {
+
+    useEffect(() => {
+        navigation.setOptions({headerShown:false});
+    }, [])
+   
+    const navigateToHabits = () => {
+
+        navigation.navigate("Habits")
+    };
+
+    return (
+        <HomeContainer source={headerImage}>
+
+            <TextContainer>
+        <TextOne>
+            Do you want to make a diffrence 
+            in you life?
+            </TextOne>
+           
+            <TextTwo>
+            With New Habits you can make life
+            as you want it!
+            </TextTwo>
+            </TextContainer>
+            <Footer>
+            <Button title="New Habits >" onPress={navigateToHabits}> </Button>
+            </Footer>
+
+        </HomeContainer>
```

Comment:
Nice job with the changes Nathalie! There are still some places that have a weird indentation, such as here 👆. Also, remember to indent styled components, e.g:
```
const Main = styled.View`
  flex:1;
  margin-bottom: 70px;
`
```
Also, remove unnecessary empty lines, and make sure you are consistent regarding semicolons.

Reply:
This still applies 👆